Algorithms/Bubblesort/python/bubbleSort.py

```diff
@@ -1,17 +1,16 @@
-#Bubble sort for an that is given.
```

Comment:
@kp7742 this file is updated with simpler code, please remove this file from the commit.

Reply:
Some Problem during Reverting of Changes, Sorry for that. I'm Closing this Pull and Requesting New